pkg/controllers/organization/dex_controller.go

```diff
@@ -51,52 +40,7 @@ type DexReconciler struct {
 //+kubebuilder:rbac:groups=dex.coreos.com,resources=connectors;oauth2clients,verbs=get;list;watch;create;update;patch
 //+kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch
 
-// SetupWithManager sets up the controller with the Manager.
-func (r *DexReconciler) SetupWithManager(name string, mgr ctrl.Manager) error {
-	r.Client = mgr.GetClient()
-	r.recorder = mgr.GetEventRecorderFor(name)
-	if r.Namespace == "" {
-		return errors.New("namespace required but missing")
-	}
-	return ctrl.NewControllerManagedBy(mgr).
-		Named(name).
-		For(&greenhousesapv1alpha1.Organization{},
-			builder.WithPredicates(clientutil.PredicateHasOICDConfigured())).
-		Owns(&dexapi.Connector{}).
-		Owns(&dexapi.OAuth2Client{}).
```

Comment:
These need to be added to the OrganizationReconciler as well.

Reply:
Done